server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -4367,14 +4368,7 @@ public boolean checkIfDynamicScalingCanBeEnabled(VirtualMachine vm, ServiceOffer
      */
     protected long configureCustomRootDiskSize(Map<String, String> customParameters, VMTemplateVO template, HypervisorType hypervisorType, DiskOfferingVO rootDiskOffering) {
         verifyIfHypervisorSupportsRootdiskSizeOverride(hypervisorType);
-        Long rootDiskSizeCustomParam = null;
```

Comment:
why we had to remove the rootDiskSizeCustomParam related code here ?

Reply:
Appears to have been a refactor for a flagged potential NPE, but reverting to avoid any regression as its not needed for the functional part of this PR.  Reverted in commit 9f8daa2878.